Index: src/Coffe.java
===================================================================
diff --git a/src/Coffe.java b/src/Coffe.java
deleted file mode 100644
--- a/src/Coffe.java	(revision 13d5bcad3dc75aac143f1546bc534a45a0761369)
+++ /dev/null	(revision 13d5bcad3dc75aac143f1546bc534a45a0761369)
@@ -1,17 +0,0 @@
-import java.util.List;
-
-public class Coffe extends Shop{
-    public Coffe(String name, String address, String numberToContact, String webSite, List<Employee> employee, DayEnum day) {
-        super(name, address, numberToContact, webSite, employee, day);
-    }
-
-    @Override
-    public String averageMarks() {
-        return null;
-    }
-
-    @Override
-    public String toString() {
-        return null;
-    }
-}
Index: src/ClothesShop.java
===================================================================
diff --git a/src/ClothesShop.java b/src/ClothesShop.java
deleted file mode 100644
--- a/src/ClothesShop.java	(revision 13d5bcad3dc75aac143f1546bc534a45a0761369)
+++ /dev/null	(revision 13d5bcad3dc75aac143f1546bc534a45a0761369)
@@ -1,30 +0,0 @@
-import java.util.List;
-
-public class ClothesShop extends Shop {
-
-    //valutare se attribuire al cliente anzi che al negozio
-    // i clienti possono avere una lista di card
-    private LoyalityCardsEnum card;
-
-
-    public ClothesShop(String name, String address, String numberToContact, String webSite, List<Employee> employee, DayEnum day) {
-        super(name, address, numberToContact, webSite, employee, day);
-        this.card=card;
-    }
-
-    @Override
-    public String averageMarks() {
-        return null;
-    }
-
-    @Override
-    public String toString() {
-        return "Name: " + getName() + "\n" +
-                "Address: " + getAddress() + "\n" +
-                "Number contact: " + getNumberToContact() + "\n" +
-                "Website: " + getWebSite() + "\n" +
-                "Employee: " + getEmployee() + "\n" +
-                "Day: " + getDay();
-    }
-}
-}
Index: src/TecnologyShop.java
===================================================================
diff --git a/src/TecnologyShop.java b/src/TecnologyShop.java
deleted file mode 100644
--- a/src/TecnologyShop.java	(revision 13d5bcad3dc75aac143f1546bc534a45a0761369)
+++ /dev/null	(revision 13d5bcad3dc75aac143f1546bc534a45a0761369)
@@ -1,22 +0,0 @@
-import java.util.List;
-
-public class TecnologyShop extends Shop{
-    //valutare se attribuire al cliente anzi che al negozio
-    // i clienti possono avere una lista di card
-    private LoyalityCardsEnum card;
-
-    public TecnologyShop(String name, String address, String numberToContact, String webSite, List<Employee> employee, DayEnum day) {
-        super(name, address, numberToContact, webSite, employee, day);
-        this.card=card;
-    }
-
-    @Override
-    public String averageMarks() {
-        return null;
-    }
-
-    @Override
-    public String toString() {
-        return null;
-    }
-}
